### ISSUE TITLE
Sickchill Python3 on Synology unable to save settings

### ISSUE THREAD

**Issue by soothingone** [SickChill/SickChill#6817]
### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master/7b98df06ad3357a377d64bf31e11db8d4e372f8f 
OS: Linux-3.10.105-x86_64-with-glibc2.2.5 (Synology)
Browser: Firefox
What you did: Trying to save web port to use 8083 which I always used prior to upgrade
What happened: Setting does not seem to save, Sickchill still running on port 8899
What you expected: Port to change to 8083

I have also tried shitting the sickchill down inside DSM, manually update the web_port to 8083, and start it again, still runs on port 8899.

Logs:
```

2020-09-29 12:52:35 DEBUG :: CHECKVERSION :: check_for_new_news: Checking GitHub for latest news.
2020-09-29 12:52:35 DEBUG :: CHECKVERSION :: cur_commit = 7b98df06ad3357a377d64bf31e11db8d4e372f8f, newest_commit = 7b98df06ad3357a377d64bf31e11db8d4e372f8f, num_commits_behind = 0, num_commits_ahead = 0
2020-09-29 12:52:35 DEBUG :: CHECKVERSION :: git rev-list --left-right "@{upstream}"...HEAD : returned successful
2020-09-29 12:52:35 DEBUG :: CHECKVERSION :: Executing git rev-list --left-right "@{upstream}"...HEAD with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:35 DEBUG :: CHECKVERSION :: git rev-parse --verify --quiet "@{upstream}" : returned successful
2020-09-29 12:52:34 DEBUG :: CHECKVERSION :: Executing git rev-parse --verify --quiet "@{upstream}" with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:34 DEBUG :: CHECKVERSION :: git branch --set-upstream-to origin/master : returned successful
2020-09-29 12:52:34 DEBUG :: CHECKVERSION :: Executing git branch --set-upstream-to origin/master with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:34 DEBUG :: CHECKVERSION :: git fetch origin --prune : returned successful
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Raised by Wolves (2020) set to: [ raised by wolves 2020 ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Raised by Wolves (2020)
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Good Doctor set to: [ the good doctor ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Good Doctor
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Space Force set to: [ space force ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Space Force
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Upload set to: [ upload ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Upload
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Undoing set to: [ the undoing ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Undoing
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for FBI: Most Wanted set to: [ fbi most wanted ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for FBI: Most Wanted
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Alienist set to: [ the alienist ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Alienist
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Legacies set to: [ legacies ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Legacies
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Mandalorian set to: [ the mandalorian ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Mandalorian
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for His Dark Materials set to: [ his dark materials ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for His Dark Materials
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Boys set to: [ the boys ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Boys
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Titans (2018) set to: [ titans 2018 ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Titans (2018)
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Euphoria (US) set to: [ euphoria us ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Euphoria (US)
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Doom Patrol set to: [ doom patrol ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Doom Patrol
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Umbrella Academy set to: [ the umbrella academy ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Umbrella Academy
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Snatch set to: [ snatch ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Snatch
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for FBI set to: [ fbi ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for FBI
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Altered Carbon set to: [ altered carbon ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Altered Carbon
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Manifest set to: [ manifest ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Manifest
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Star Trek: Discovery set to: [ star trek discovery ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Star Trek: Discovery
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Deep State set to: [ deep state ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Deep State
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Handmaid's Tale set to: [ the handmaids tale ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Handmaid's Tale
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Grand Tour (2016) set to: [ the grand tour, the grand tour 2016 ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Grand Tour (2016)
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Westworld set to: [ westworld ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Westworld
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Billions set to: [ billions ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Billions
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for DC's Legends of Tomorrow set to: [ dcs legends of tomorrow ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for DC's Legends of Tomorrow
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for Lucifer set to: [ lucifer ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for Lucifer
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The Flash (2014) set to: [ the flash 2014 ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The Flash (2014)
2020-09-29 12:52:34 DEBUG :: MAIN :: Internal name cache for The 100 set to: [ the 100 ]
2020-09-29 12:52:34 DEBUG :: MAIN :: Building internal name cache for The 100
2020-09-29 12:52:33 DEBUG :: MAIN :: Internal name cache for Vikings set to: [ vikings ]
2020-09-29 12:52:33 DEBUG :: MAIN :: Building internal name cache for Vikings
2020-09-29 12:52:33 DEBUG :: MAIN :: Internal name cache for The Blacklist set to: [ the blacklist ]
2020-09-29 12:52:33 DEBUG :: MAIN :: Building internal name cache for The Blacklist
2020-09-29 12:52:33 DEBUG :: CHECKVERSION :: Executing git fetch origin --prune with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:33 DEBUG :: CHECKVERSION :: git config remote.origin.url https://github.com/SickChill/SickChill.git : returned successful
2020-09-29 12:52:33 DEBUG :: MAIN :: Internal name cache for The Walking Dead set to: [ the walking dead ]
2020-09-29 12:52:33 DEBUG :: CHECKVERSION :: Executing git config remote.origin.url https://github.com/SickChill/SickChill.git with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:33 DEBUG :: CHECKVERSION :: git rev-parse HEAD : returned successful
2020-09-29 12:52:33 DEBUG :: MAIN :: Building internal name cache for The Walking Dead
2020-09-29 12:52:33 DEBUG :: CHECKVERSION :: Executing git rev-parse HEAD with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:33 DEBUG :: CHECKVERSION :: git symbolic-ref -q HEAD : returned successful
2020-09-29 12:52:33 DEBUG :: CHECKVERSION :: Executing git symbolic-ref -q HEAD with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:33 INFO :: CHECKVERSION :: Checking for updates using GIT
2020-09-29 12:52:33 DEBUG :: CHECKVERSION :: Starting new thread: CHECKVERSION
2020-09-29 12:52:33 INFO :: WEBSERVER :: Starting SickChill on http://0.0.0.0:8899/
2020-09-29 12:52:33 INFO :: MAIN :: Forcing web server to port 8899
2020-09-29 12:52:33 INFO :: MAIN :: Starting SickChill [master] using '/volume1/@appstore/sickchill/var/config.ini'
2020-09-29 12:52:33 DEBUG :: MAIN :: Loading initial show list
2020-09-29 12:52:33 DEBUG :: MAIN :: Retrieving the last check time from the DB
2020-09-29 12:52:33 DEBUG :: MAIN :: git symbolic-ref -q HEAD : returned successful
2020-09-29 12:52:33 DEBUG :: MAIN :: Executing git symbolic-ref -q HEAD with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:33 DEBUG :: MAIN :: Using: git
2020-09-29 12:52:33 DEBUG :: MAIN :: git version : returned successful
2020-09-29 12:52:33 DEBUG :: MAIN :: Executing git version with your shell in /volume1/@appstore/sickchill/var/src/sickchill
2020-09-29 12:52:33 DEBUG :: MAIN :: Checking if we can use git commands: git version
2020-09-29 12:52:33 DEBUG :: MAIN :: Checking for archived episodes not qualified
2020-09-29 12:52:33 DEBUG :: MAIN :: Checking database structure...failed.db
2020-09-29 12:52:33 DEBUG :: MAIN :: Checking database structure...cache.db
2020-09-29 12:52:33 DEBUG :: MAIN :: Checking database structure...sickchill.db
2020-09-29 12:52:32 DEBUG :: MAIN :: The name Show Name - 2x03 - Ep Name.ext parsed into Show Name - S02E03 [ABD: False] [ANIME: False] [whichReg: ['anime SxEE']] Score: -1
2020-09-29 12:52:32 DEBUG :: MAIN :: Parsed Show Name - 2x03 - Ep Name.ext into Show Name - S02E03 [ABD: False] [ANIME: False] [whichReg: ['anime SxEE']] Score: -1
2020-09-29 12:52:32 DEBUG :: MAIN :: Trying to parse Show Name - 2x03 - Ep Name.ext
2020-09-29 12:52:32 DEBUG :: MAIN :: formatting pattern: Season %0S -> Season 02
2020-09-29 12:52:32 DEBUG :: MAIN :: Found codec for 'Show Name: Ep Name'.
2020-09-29 12:52:32 DEBUG :: MAIN :: Parsed Show.Name.S02E03.HDTV.XviD-SICKCHILL into Show Name - S02E03 [GROUP: SICKCHILL] [ABD: False] [ANIME: False] [whichReg: ['anime_SxxExx']] Score: -2
2020-09-29 12:52:32 DEBUG :: MAIN :: formatting pattern: %SN - %Sx%0E - %EN -> Show Name - 2x03 - Ep Name
2020-09-29 12:52:32 DEBUG :: MAIN :: Found codec for 'Show Name: Ep Name'.
2020-09-29 12:52:32 DEBUG :: MAIN :: Parsed Show.Name.S02E03.HDTV.XviD-SICKCHILL into Show Name - S02E03 [GROUP: SICKCHILL] [ABD: False] [ANIME: False] [whichReg: ['anime_SxxExx']] Score: -2
2020-09-29 12:52:25 INFO :: EVENT-QUEUE :: Restarting SickChill with ['/var/packages/python3/target/bin/python3', '/volume1/@appstore/sickchill/var/src/SickChill.py', '--daemon', '--pidfile=/volume1/@appstore/sickchill/var/sickchill.pid', '--datadir=/volume1/@appstore/sickchill/var', '--port=8899', '--nolaunch']

```

**Comment by miigotu** [comment]
Because you used the wrong package. We do not support the benjv package that we have no changelog or git history for. It set your config and data to the root user.

We have an official package.
https://github.com/SynoCommunity/spksrc/pull/4161
Port is 8081.

**Comment by caramdache** [comment]
Thanks, I upgraded from benvj to the synocommunity package.

Ran into a couple of issues however:

- could not use the Synology Python3 package, had to install the SynoCommunity one instead;
- could not import the backup from the benjv package (SickChill would never restart); had to diff (and sort) the config.ini files to reapply the config manually;
- had to browse sickbeard.db/table tv_shows to determine which shows were previously registered.

The latter was rather tedious. An easier export/import process would help.

Did I miss something obvious?
